README.md

```diff
@@ -126,7 +126,8 @@ define a `remote` key. Only 1 level of remote configuration is supported by desi
 
 ```yaml
 # The remote repository to read the policy file from. This is required, and must
-# be in the form of "org/repo-name". Must be a public repository.
+# be in the form of "org/repo-name". The policy bot github app must have read
+# access to this repository.
```

Comment:
i added this because it was little misleading

Reply:
Thanks, that's probably a bit clearer... hopefully it is implied that apps have read access to public repositories.